pkg/scripts/funcs.go

```diff
@@ -79,6 +79,10 @@ var (
 		EOF
 		sudo systemctl force-reload systemd-journald
 		{{ end }}
+
+        {{ define "load-iptables-modules" }}
+        modprobe ip_tables
```

Comment:
Why not `sudo modprobe`? We usually SSH as a non-root user and then use `sudo` (see other commands).